src/api/services/authService.ts

```diff
@@ -0,0 +1,9 @@
+const login = async () => {
+
+}
+
+const authService = {
+  login
+};
+
+export default authService;
```

Comment:
Still using?

Reply:
No, I will remove it.

Reply:
Based on the other suggestions I give you after, you can use it and add it methods